modules/ztp-about-ztp-and-single-node-openshift-clusters-factory.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
```

Comment:
Preview link for wherever this is?

Reply:
My bad, I initially started with this as part of content, but removed. I should have deleted these files and updated the PR accordingly, sorry about that.